can/parser_pyx.pyx

```diff
@@ -36,42 +36,39 @@ cdef class CANParser:
   def __init__(self, dbc_name, signals, checks=None, bus=0, enforce_checks=True):
     if checks is None:
       checks = []
-    self.can_valid = True
     self.dbc_name = dbc_name
     self.dbc = dbc_lookup(dbc_name)
     if not self.dbc:
       raise RuntimeError(f"Can't find DBC: {dbc_name}")
     self.vl = {}
     self.ts = {}
-
+    self.can_valid = True
     self.can_invalid_cnt = CAN_INVALID_CNT
 
-    cdef int i
-    cdef int num_msgs = self.dbc[0].num_msgs
-    for i in range(num_msgs):
```

Comment:
Didn't see any noticeable differences with the generated code with/without the cdef: https://www.diffchecker.com/Vmp4EWm2.